skin/frontend/base/default/js/oyst/cart/oneclick.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * This file is part of Oyst_OneClick for Magento.
+ *
+ * @license http://www.apache.org/licenses/LICENSE-2.0 Apache License, Version 2.0
+ * @author Oyst <plugin@oyst.com> <@oyst>
+ * @category Oyst
+ * @package Oyst_OneClick
+ * @copyright Copyright (c) 2017 Oyst (http://www.oyst.com)
+ */
+
+/**
+ * OneClick Customs JS
+ */
+"use strict";
+
+/**
+ * Main function to init 1-Click
+ *
+ * @param {String} oneClickUrl  Backend conf oneclick payment url
+ */
+function oystOneClick(oneClickUrl) {
+    window.__OYST__ = window.__OYST__ || {};
+    window.__OYST__.getOneClickURL = function (cb, opts) {
+        opts = opts || {};
+
+        ready(function () {
+            var form = new FormData();
+            form.append("preload", opts.preload);
+
+            var products = [];
+            var actions = document.getElementsByClassName("product-cart-actions");
+
+            for (var i = 0; i < actions.length; i++) {
```

Comment:
use `actions.forEach(...`

Reply:
Can't use forEach because it's an HTMLCollection

Reply:
Ok ;)